src/Microsoft.Android.Sdk.ILLink/LinkContextExtensions.cs

```diff
@@ -0,0 +1,161 @@
+#nullable enable
+
+using Mono.Cecil;
+using Mono.Collections.Generic;
+using Mono.Linker;
+using System.Collections.Generic;
+
+namespace Microsoft.Android.Sdk.ILLink
+{
+    public static class LinkContextExtensions
+    {
+        public static MethodDefinition GetBaseDefinition (this LinkContext context, MethodDefinition method)
+		{
```

Comment:
There is some weird indentation here. You might run `dotnet format` on this file, so it will follow our `.editorconfig`.

Reply:
Oops, fixed. Thanks